util/src/main/java/tech/pegasys/artemis/util/cli/VersionProvider.java

```diff
@@ -45,6 +45,16 @@ public static String detectJvm() {
     return detectedVM + "-java-" + detectedJavaVersion;
   }
 
+  public static String defaultStoragePath() {
+    final String detectedOS = normalizeOS(normalize("os.name"));
+    if (detectedOS.equals("windows")) {
+      return System.getenv("LOCALAPPDATA") + "/teku";
+    } else if (detectedOS.equals("osx")) {
+      return System.getenv("HOME") + "/Library/teku";
+    }
+    return "~/.teku";
```

Comment:
This should be the `$XDG_DATA_HOME`:

```suggestion
    return "~/.local/share/teku";
```

> `$XDG_DATA_HOME` defines the base directory relative to which user specific data files should be stored. If `$XDG_DATA_HOME` is either not set or empty, a default equal to `$HOME/.local/share` should be used. 

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

> Historically, Unix programs were free to spread their data all over the `$HOME` directory, putting their data in dot-files (files starting with ".") or subdirectories such as `~/.vimrc` and `~/.vim`. The new specification is intended to make this behavior more predictable.

https://askubuntu.com/questions/14535/whats-the-local-folder-for-in-my-home-directory

Reply:
Thanks for the feedback, I appreciate the level of information you've given, it's a compelling argument. I've made a larger change to incorporate your suggestion so that I can test the behaviors and make sure they're what is expected.